starter-code/index.js

```diff
@@ -1,10 +1,35 @@
-var SortedList = function() {};
+var SortedList = function() {
+  this.items = [];
+  this.length = 0;
+};
 
-SortedList.prototype.add     = function(item) {}
-SortedList.prototype.get     = function(pos) {}
-SortedList.prototype.max     = function() {}
-SortedList.prototype.min     = function() {}
-SortedList.prototype.average = function() {}
-SortedList.prototype.sum     = function() {}
+SortedList.prototype.add = function(item) {
+  this.items.push(item);
+  this.length = this.items.length;
+  this.items.sort(function(a, b) {
```

Comment:
Now with ES6 syntax this could be shortened to a nice:
`this.items.sort( (a, b) => a - b )`